frontend/packages/console-dynamic-plugin-sdk/src/utils/fetch/co-fetch-utils.ts

```diff
@@ -0,0 +1,118 @@
+import * as _ from 'lodash';
+import { authSvc } from '@console/internal/module/auth';
+import { RetryError, HttpError } from '../error/http-error';
+import { InternalReduxStore } from '../redux';
+
+const cookiePrefix = 'csrf-token=';
+export const getCSRFToken = () =>
```

Comment:
Just confirming that the utilities in this file are only used internally and are not public API.

Reply:
Yes, they are only used internally with an exception of `getImpersonateHeaders` which is being exposed for monorepo packages to consume. Only `coFetch`, `coFetchJSON` and `coFetchText` are part of public API of SDK.